ros_ign_bridge/test/utils/ign_test_msg.hpp

```diff
@@ -0,0 +1,297 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef UTILS__IGN_TEST_MSG_HPP_
+#define UTILS__IGN_TEST_MSG_HPP_
+
+#include <ignition/msgs.hh>
```

Comment:
Let's break this up into the actual `pb.h`s needed, that should help compilation times even more